### DISCUSSION TITLE
[Feature Suggestion] Add an CLI option to detect + retry failures from previous run

### DISCUSSION THREAD

**Discussion by stevenaw** [nunit/nunit-console#1528] [Ideas]
An interesting idea some other test frameworks/runners offer is the ability to retry failures from a previous run. It might be helpful to support a command line switch which, when provided, will look for and parse a previous run's "result" spec and re-run the failed tests contained within. It could then rewrite to the same file, overwriting the previous contents (enabling potentially multiple levels of retries through successive runs if desired)

This could be a way to support the "assembly-level retry" feature requested within the framework in https://github.com/nunit/nunit/issues/2452

**Comment by rprouse** [comment]
We could probably do this with a simple text file with a list of failed tests one per line then plug it into our existing filtering.
